crates/stc_ts_file_analyzer/src/analyzer/expr/mod.rs

```diff
@@ -333,15 +333,6 @@ impl Analyzer<'_, '_> {
         ty.freeze();
         self.cur_facts.assert_clone_cheap();
 
-        if !self.config.is_builtin && !ty.is_any() {
```

Comment:
This should not be removed.

Reply:
Please change the span of the generated `Type` instead

Reply:
span of builtin type eqaul dummy_span
chnage to span of `import.meta` keyword?
@kdy1

Reply:
Yes